minio/api.py

```diff
@@ -2002,10 +2002,14 @@ def remove_objects(self, bucket_name, delete_object_list,
                 print("error occurred when deleting object", error)
 
             # Remove a prefix recursively.
-            delete_object_list = map(
-                lambda x: DeleteObject(x.object_name),
-                client.list_objects("my-bucket", "my/prefix/", recursive=True),
-            )
+            delete_object_list = [
+                DeleteObject(x).object_name
+                for x in client.list_objects(
```

Comment:
What problem did you get when using `map()`?

Reply:
`list(map(...))` works fine, but just using `map` doesn't work with `Minio.remove_objects`. This method expects a list. Passing a `map` fails silently and the objects are not removed.

In the above commits, I changed from `list(map(...))` to a list comprehension to make the code a bit more concise.

Reply:
That is too verbose. Please make `list(map())`

Reply:
Reverted to `list(map())`